Lab-1/README.md

```diff
@@ -26,85 +26,101 @@ In this tutorial, you'll learn how to build an analysis solution by bringing rea
-
-3.	Click **STREAM ANALYTICS** in the left pane to list the Stream Analytics jobs.
-
-4.	The new job will be shown with a status of **CREATED**. Notice that the **START** button on the bottom of the page is disabled. You must configure the job input, output, and query before you can start the job.
-
-
-### Specify job input
-1.	In your Stream Analytics job click **INPUTS** from the top of the page, and then click **ADD INPUT**. The dialog that opens will walk you through a number of steps to set up your input.
-1.	Select **DATA STREAM**, and then click the right button.
-1.	Select **EVENT HUB**, and then click the right button.
-1.	Type or select the following values on the third page:
-
-* **INPUT ALIAS**: Enter a friendly name for this job input, such as TwitterStream. Note that you will be using this name in the query later on.
-**EVENT HUB**: If the Event Hub you created is in the same subscription as the Stream Analytics job, select the namespace that the event hub is in.
+1. Navigate to the azure management portal and sign in to it 
+2. Select “New” from the bottom of the screen and Navigate to “Data Services”, “Stream Analytics”, “Quick Create”
+3. Provide “Job Name”, example “bddemostream”, select the “Region” and either provide an existing storage account or create a new storage account to store monitoring data for the stream. Ensure that all resources are in the same region to avoid network latency and data transfer charges. 
+4. Once the stream is created, we will need to configure it to define the input and output, as well as queries before the job is started. 
+5. Select the newly created stream – “bddemostream” to navigate to the options for the stream. 
+6. Select “Inputs” from the menu options and then click on “Add Input” to provide the input to the stream. 
+7. Select “Data Stream” from the options provided to define input to the stream 
+8. Select “Event Hub” from the options provided 
+9. Provide an “Input Alias”, example “inputtweets” and select the “Event Hub Namespace”, “Event Hub”, “Event Hub Policy” and choose the “Consumer Group” 
+10. Navigate to the next screen by clicking on the “Right arrow” at the bottom of the screen 
+11. Select the “Event Serialization Format” as “JSON” and the “Encoding” as “UTF8”, by default these are the selections, we can leave it at that. 
+12. Select the tick mark on the bottom to complete the creation of the Input. 
+13. Select the option “Test Connection” from the bottom menu bar to verify the connection to the Event Hub
 
-	If your event hub is in a different subscription, select **Use Event Hub from Another Subscription**, and then manually enter information for **SERVICE BUS NAMESPACE**, **EVENT HUB NAME**, **EVENT HUB POLICY NAME**, **EVENT HUB POLICY KEY**, and **EVENT HUB PARTITION COUNT**.
```

Comment:
Just curious - is this going to cause any issues, with this information about different subscriptions removed?